contracts/proxy-lib/src/ext_config.rs

```diff
@@ -0,0 +1,11 @@
+use near_sdk::ext_contract;
+use calimero_context_config::repr::Repr;
+use calimero_context_config::types::{SignerId, ContextId};
+
+#[ext_contract(config_contract)]
+trait Configcontract {
+    fn has_member(&self,
```

Comment:
has this been implemented in the ctx config contract?

Reply:
here it goes https://github.com/calimero-network/core/pull/854/files#diff-22355c320575851613e81b59302954d94433115e28c09199b70079772934a535R47

Reply:
Ah, missed that, noice.